### ISSUE TITLE
Reconsider definition of <project_group>

### ISSUE THREAD

**Issue by sophie-h** [ximion/appstream#429]
The [current definition](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-project_group) of `<project_group>` sounds somewhat similar to what in GNOME is called "Core apps" or "official GNOME software."

A quick check of Flathub gave me the following usage stats for `<project_group>`

```
GNOME:    167
KDE:      144
Pop!_OS:  1
Qt:       7
Solarus:  1
XFCE:     2
```

At the same time, there currently exist only 29 Core apps in GNOME and only 20 are on Flathub for technical reasons.

I don't think it's realistic to roll back time with the use of this tag. I would suggest one of the following

1. Adapt the meaning of the tag in the direction of "fits into the ecosystem of KDE, GNOME, ..."
2. Deprecate the tag and create new tags for 'part of ecosystem' and 'official XXX app'

But those are just some quick ideas

/cc @bertob

Is there someone from KDE we can ping?

**Comment by aleixpol** [comment]
> Is there someone from KDE we can ping?

Hello :)

Thanks for bringing this up, I agree it makes sense. At the moment we are not using this information at all in Plasma. I wouldn't be heartbroken if it was deprecated either.

At the moment we are using the appstream id anyway to see where an app is coming from (i.e. org.kde.kate is from KDE, org.gnome.gedit is gnome's).

**Comment by bertob** [comment]
I agree this needs to be cleared up, but no strong opinion on the exact path forward. Both of your suggestions sound good to me :)

**Comment by sophie-h** [comment]
For context: The most prominent use of `<project_group>` would probably be this feature that is currently part of Flathub beta

![image](https://user-images.githubusercontent.com/3466497/186686483-3a4e904a-6438-42be-977e-09946d913ae6.png)

**Comment by ximion** [comment]
Looks like people aren't validating their metadata - I think the validator will emit an info hint if people are in e.g. project group "KDE" but don't have their ID start with "org.kde.*".
The intent of this tag was always to assign programs to an umbrella project, so e.g. GNOME Circle apps or anything that is tied to GNOME would fall under that umbrella. Same with KDE, anything that the KDE community developers would be a KDE project, while anything that uses Qt would not be.
In the same way, the project group "Mozilla" would only be for apps that are produced by the Mozilla organization, and not random people who just want to develop stuff for Firefox.

This feature actually used by software centers to rank e.g. GNOME applications higher when running on GNOME, and KDE applications higher when running on KDE (that was what we added this for originally).

I am not sure if "part of the ecosystem" is such a great idea here, because many apps would fit into multiple ecosystems and people may feel pressured into just putting one of the values in, resulting in ranking differences in search results and other unintended consequences (can anyone claim to be a GNOME Member or be part of KDE, even for a random proprietary app?). We could change the wording, but I think it would still be good to keep the original intent of the tag, which in abstract terms is "machine-readable, untranslatable string to associate this software components with a particular larger organization or collective that is making software, like Freedesktop, KDE, GNOME, Mozilla, ...". This is different from `developer_name` which is the actual entity that created the software (which *may* also be a value like "KDE e.V." but may also just be an individual developer, or a corporation like Microsoft).
